data/apply-planet_osm_line.sql

```diff
@@ -32,31 +32,54 @@ UPDATE planet_osm_line
 UPDATE planet_osm_line
   SET mz_label_placement = ST_PointOnSurface(way);
 
-CREATE INDEX planet_osm_line_roads_geom_index ON planet_osm_line USING gist(way) WHERE mz_road_level IS NOT NULL;
-CREATE INDEX planet_osm_line_roads_geom_9_index ON planet_osm_line USING gist(way) WHERE mz_road_level <= 9;
-CREATE INDEX planet_osm_line_roads_geom_12_index ON planet_osm_line USING gist(way) WHERE mz_road_level <= 12;
-CREATE INDEX planet_osm_line_roads_geom_15_index ON planet_osm_line USING gist(way) WHERE mz_road_level <= 15;
-
-CREATE INDEX planet_osm_line_transit_geom_index ON planet_osm_line USING gist(way) WHERE mz_transit_level IS NOT NULL;
-CREATE INDEX planet_osm_line_transit_geom_6_index ON planet_osm_line USING gist(way) WHERE mz_transit_level <= 6;
-CREATE INDEX planet_osm_line_transit_geom_9_index ON planet_osm_line USING gist(way) WHERE mz_transit_level <= 9;
-CREATE INDEX planet_osm_line_transit_geom_12_index ON planet_osm_line USING gist(way) WHERE mz_transit_level <= 12;
-CREATE INDEX planet_osm_line_transit_geom_15_index ON planet_osm_line USING gist(way) WHERE mz_transit_level <= 15;
-
-CREATE INDEX planet_osm_line_earth_geom_index ON planet_osm_line USING gist(way) WHERE mz_earth_min_zoom IS NOT NULL;
-CREATE INDEX planet_osm_line_earth_geom_9_index ON planet_osm_line USING gist(way) WHERE mz_earth_min_zoom <= 9;
-CREATE INDEX planet_osm_line_earth_geom_12_index ON planet_osm_line USING gist(way) WHERE mz_earth_min_zoom <= 12;
-CREATE INDEX planet_osm_line_earth_geom_15_index ON planet_osm_line USING gist(way) WHERE mz_earth_min_zoom <= 15;
-
-CREATE INDEX planet_osm_line_water_geom_index ON planet_osm_line USING gist(way) WHERE mz_water_min_zoom IS NOT NULL;
-CREATE INDEX planet_osm_line_water_geom_9_index ON planet_osm_line USING gist(way) WHERE mz_water_min_zoom <= 9;
-CREATE INDEX planet_osm_line_water_geom_12_index ON planet_osm_line USING gist(way) WHERE mz_water_min_zoom <= 12;
-CREATE INDEX planet_osm_line_water_geom_15_index ON planet_osm_line USING gist(way) WHERE mz_water_min_zoom <= 15;
-
-CREATE INDEX planet_osm_line_boundary_geom_index ON planet_osm_line USING gist(way) WHERE mz_boundary_min_zoom IS NOT NULL;
-CREATE INDEX planet_osm_line_boundary_geom_9_index ON planet_osm_line USING gist(way) WHERE mz_boundary_min_zoom <= 9;
-CREATE INDEX planet_osm_line_boundary_geom_12_index ON planet_osm_line USING gist(way) WHERE mz_boundary_min_zoom <= 12;
-CREATE INDEX planet_osm_line_boundary_geom_15_index ON planet_osm_line USING gist(way) WHERE mz_boundary_min_zoom <= 15;
+-- only these 2 columns are relevant in lower zoom queries
```

Comment:
The pois layer needs to accompany any landuse layer early start_zoom (so labels can be shown for the polygons).

Reply:
If we just have these two columns as part of the query, is Postgres able to use the index to accelerate it? I'm not sure how "clever" it is at knowing that the index is a superset of these requirements.

Reply:
I wasn't sure, so I opted to make the index match the query exactly here. To make sure I understand, are you suggesting keeping all the columns in the index where clauses, even if they aren't used, for consistency purposes? Assuming that postgres would continue to use the index, which I can verify.

Reply:
Ah, no, sorry. I had misunderstood. What you've described sounds right, I just wanted to make sure that our indexes are the same "shape" as our queries. (Or that we're verifying Postgres can handle the different "shapes".)